Linear_cell_complex/test/Linear_cell_complex/flat.cpp

```diff
@@ -0,0 +1,12 @@
+
+#include <boost/property_tree/xml_parser.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include <boost/container/flat_set.hpp>
+
+int main()
+{
+  boost::container::flat_set<int> fs;
+
+  return 0;
+}
```

Comment:
What is the plan for that file? We thought it was a minimal test case for the bug, but it is not.

Actually, all the docker images have been rebuild two days ago, and the version of Boost has changed. Maybe the bug is already fixed.

I suggest the file to be removed.